.github/workflows/marketplace-publish.yml

```diff
@@ -51,3 +53,10 @@ jobs:
 
           npm run publish:marketplace
           echo "Successfully published version $current_package_version to VS Code Marketplace"
+      
+      - name: Create and Push Git Tag
```

Comment:
Maybe move this just before publishing the extension?

Reply:
What do you think is best?

Reply:
I'd say tagging before publishing is more common.

Reply:
Works for me! Want to update it and then I can merge?

Reply:
I've moved tagging before publishing, but still after packaging to avoid tagging releases which fail to package.